### DISCUSSION TITLE
Problem upgrading to latest frigate .11 release

### DISCUSSION THREAD

**Discussion by stamandr** [blakeblackshear/frigate#3178] [General]
I am currently using blakeblackshear/frigate:release-0.11.0-cbf26e0 on a RPI 4.  I am wanting to try the latest version, which I believe is blakeblackshear/frigate:release-0.11.0-3de1948. I have added this to my docker compose but get the following error

root@raspberrypi4:~# docker-compose -f frigate_docker-compose.yml up -d
Pulling frigate (blakeblackshear/frigate:release-0.11.0-3de1948)...
**ERROR: manifest for blakeblackshear/frigate:release-0.11.0-3de1948 not found: manifest unknown: manifest unknown**

Any suggestions ?

**Comment by NickM-27** [comment]
There's been a single version of 0.11 beta released specifically for r pi users (same as 0.10 with hwaccel enabled). Other than that 0.11 has not been released as a beta and should not be used / downloaded

**Reply by stamandr** [reply]
So the only version is, release-0.11.0-cbf26e0 to address the hwaccel issue with .10 ?

**Reply by NickM-27** [reply]
> So the only version is, release-0.11.0-cbf26e0 to address the hwaccel issue with .10 ?

Correct. An actual 0.11 beta will be released soonish
